Connectivity

LenSpek opened the pull request:
When I was checking my work I noticed that I forgot to add some information about connectivity, so it is added now

maedoc commented:
thank you

+1 merge unless you'll continue to push commits to the PR

JohnGriffiths commented:
Go ahead. 

I have some more similar PRs coming from a different branch shortly.

LenSpek committed:
Connectivity

When I was checking my work I noticed that I forgot to add some information about connectivity, so it is added now

liadomide commented:
Please get the updates from Master, before applying your changes.
In the current form, there is a conflict.

maedoc commented:
@LenSpek merge conflicts with notebooks are annoying but you might find a tool ike [nbdime](https://github.com/jupyter/nbdime) helpful.  if you aren't up to do this, let me know, and I'll handle it.
